frontend/packages/dev-console/src/components/helm/details/HelmReleaseDetails.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { match as RMatch } from 'react-router';
+import {
+  navFactory,
+  LoadingBox,
+  StatusBox,
+  FirehoseResult,
+} from '@console/internal/components/utils';
+import { SecretModel } from '@console/internal/models';
+import { ErrorPage404 } from '@console/internal/components/error';
+import { DetailsPage } from '@console/internal/components/factory';
+import { K8sResourceKindReference } from '@console/internal/module/k8s';
+import { Status } from '@console/shared';
+import { Badge } from '@patternfly/react-core';
+import { fetchHelmReleases } from '../helm-utils';
+import HelmReleaseResources from './resources/HelmReleaseResources';
+import HelmReleaseOverview from './overview/HelmReleaseOverview';
+import HelmReleaseHistory from './history/HelmReleaseHistory';
+import { deleteHelmRelease, upgradeHelmRelease } from '../../../actions/modify-helm-release';
+import HelmReleaseNotes from './HelmReleaseNotes';
+import { HelmRelease } from '../helm-types';
+
+const SecretReference: K8sResourceKindReference = 'Secret';
+const HelmReleaseReference = 'HelmRelease';
+export interface HelmReleaseDetailsProps {
+  match: RMatch<{
+    ns?: string;
+    name?: string;
+  }>;
+  secret?: FirehoseResult;
+}
+
+export interface LoadedHelmReleaseDetailsProps extends HelmReleaseDetailsProps {
```

Comment:
nit: can avoid export and will it make sense to have `LoadedHelmReleaseDetails` in diff file and have `HelmReleaseDetails` here

Reply:
The components are too tightly coupled to have it in diff files IMO.